tslint.json

```diff
@@ -6,7 +6,8 @@
   "jsRules": {},
   "rules": {
     "interface-name": [true, "never-prefix"],
-    "no-var-requires": false
+    "no-var-requires": false,
+    "ordered-imports": false
```

Comment:
A good way to do this rather than disabling it is to add [import-sort](https://github.com/renke/import-sort) as a pre-commit hook (via `husky` + `lint-staged`) to automatically sort all imports.